bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -472,40 +432,34 @@ public Object getValueAt(Object node, int columnIndex) {
     }
 
     // Return the world coordinates for the given target
-    private static Option<WorldCoords> getWorldCoords(SPTarget tp, Option<Long> when) {
+    private static Option<WorldCoords> getWorldCoords(final SPTarget tp, final Option<Long> when) {
```

Comment:
I don't see why not, but I would have to ask a couple of questions first. For example, the distance as displayed in the target editor represents what? The **angular distance** between the base position and the target?

I'm also not entirely sure how to use the `when` parameter with regards to `Coordinates`; right now, it is completely ignored in the `ITarget` anyways, so I suspect we could just ignore it entirely without issue and use `Coordinates` without worrying about it, but I'm not sure if that is right. Advice, anyone? @andrewwstephens or @tpolecat ?